ckanext/dcat/controllers.py

```diff
@@ -32,10 +32,15 @@ def read_catalog(self, _format=None):
         if not _format:
             return HomeController().index()
 
+        _profiles = toolkit.request.params.get('profiles')
```

Comment:
Does it really make sense to ask for more than one profile at the time? Can you give an example? IIRC profiles used inheritance to extend them.

Reply:
Turns out I implemented it exactly like this at the lower level :upside_down_face: Ignore me!